### ISSUE TITLE
How to use inline editing

### ISSUE THREAD

**Issue by wash34000** [ypnos-web/cakephp-datatables#80]
Hello,

Do you know if it's possible to use inline editing with your project ?
https://editor.datatables.net/examples/inline-editing/simple

**Comment by ypnos-web** [comment]
Yes, you can enable inline editing with the datatables. However note that this plugin only deals with GET requests for which the component offers the `find()` method. The POST requests from DataTables are just typical AJAX requests but you need to write the handling in your controller.

**Comment by wash34000** [comment]
Do you have some clue to enable inline editing ?
